virt_who/register.py

```diff
@@ -865,6 +883,11 @@ def satellite_hosts_get(self, ssh, register_config, host_name, host_uuid, desc="
             logger.info("Failed to get host info")
             return None
 
+    def satellite_host_display(self, ssh, register_config, host_name, host_uuid, host_hwuuid=None,
+                               desc="get host display name"):
+        output = self.satellite_hosts_get(ssh, register_config, host_name, host_uuid, host_hwuuid, desc)
+        return output['name']
+
```

Comment:
Maybe this function is not so necessary, we can directly use ```self.satellite_hosts_get()['name']```

Reply:
Yes, I agree with you. Actually, I hesitated whether I should add this. :)